Source/WebInspectorUI/UserInterface/Views/ComputedStyleDetailsSidebarPanel.js

```diff
@@ -40,6 +40,8 @@ WI.ComputedStyleDetailsSidebarPanel = class ComputedStyleDetailsSidebarPanel ext
             return;
 
         WI.detailsSidebar.selectedSidebarPanel = styleRulesPanel;
-        styleRulesPanel.panel.scrollToSectionAndHighlightProperty(property);
+        setTimeout(() => {
+            styleRulesPanel.panel.scrollToSectionAndHighlightProperty(property);
+        });
```

Comment:
`WI.SpreadsheetRulesStyleDetailsPanel.prototype.computedStyleDetailsPanelShowProperty` already has logic that should handle if it's not yet constructed (or in the middle of a layout).  is there some reason that that doesn't work (anymore)?

my guess is that inside `WI.SpreadsheetRulesStyleDetailsPanel.prototype.nodeStylesRefreshed` we clear `_propertyToSelectAndHighlight`, but it's possible that even then we may still `!this.isAttached || this.layoutPending`, so id probably change that to be something like this
```js
if (this._propertyToSelectAndHighlight) {
    let property = this._propertyToSelectAndHighlight;
    this._propertyToSelectAndHighlight = null;
    this.scrollToSectionAndHighlightProperty(property);
}
```

id much prefer solving this there rather than expecting every caller to use `setTimeout`/`requestAnimationFrame` :)

Reply:
Sorry, I didn't realise this scenario was already intended to be handled. I've found the root causes of this behaviour and fixed those instead.

> my guess is that inside WI.SpreadsheetRulesStyleDetailsPanel.prototype.nodeStylesRefreshed we clear _propertyToSelectAndHighlight, but it's possible that even then we may still !this.isAttached || this.layoutPending, so id probably change that to be something like this

Unless there's something I'm missing, that doesn't do anything. The inspector correctly determines the property to select (at least according to my debugger breakpoints), there's just a variety of factors that stop it actually jumping to it.